objects/orb-blue/config.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  animations: {
+    idle: {
+      frames: [0, 1, 2, 3, 4, 5, 6, 7],
+      frameRate: 8,
+    },
+  },
+  spriteSheets: {
+    twilioquestOrbBlue: {
+      fileName: "Orb_Blue.png",
+      frameDimensions: {
+        width: 24,
+        height: 24,
+      },
+    },
+  },
+  properties: {
+    sprite: {
+      defaultFrameIndex: 0,
+      spriteSheet: "twilioquestOrb_Blue",
```

Comment:
Should this be`twilioquestOrbBlue` -- without the underscore, so that it matches the spriteSheet property name on line 9? I'm using [this example](https://twilioquest.github.io/extension-docs/guide/custom_objects.html#our-final-custom-object-config) for reference. There are a few more instances of this ([orb_green](https://github.com/TwilioQuest/twilioquest-base/pull/83/files#:~:text=twilioquestOrb_Green), [orb_orange](https://github.com/TwilioQuest/twilioquest-base/pull/83/files#:~:text=twilioquestOrb_Orange), [orb_purple](https://github.com/TwilioQuest/twilioquest-base/pull/83/files#:~:text=twilioquestOrb_Purple), etc). I'm not actually sure if this is something to worry about, but thought I'd mention it just in case. If not, then feel free to ignore this!